ETC.tsv

```diff
@@ -6235,7 +6235,7 @@ ETC_20150317_006235	An armor made of surprisingly inexpensive materials.
 ETC_20150317_006236	Companion Full Plate Armor
 ETC_20150317_006237	It's made so it won't obstruct sight or movement.
 ETC_20150317_006238	{S18} {#001100} When a quest is complete, click on the Return button to teleport to the quest NPC. {/}
-ETC_20150317_006239	{S18} {#001100} by using the store you can buy the item. {/}
```

Comment:
this could be a line that forms a sentence with other lines. It'd be best to leave it uncapitalized.

>you can buy items